README.md

```diff
@@ -78,43 +76,32 @@ git remote add upstream https://github.com/jwplayer/jwplayer
 2. Install the dependencies:
 ````bash
 # Install dependencies
-npm install -g eslint grunt-cli jsdoc karma-cli stylelint webpack webpack-cli
 npm install
-# Optionally, install webpack-dev-server
-npm install -g webpack-dev-server
 ````
- 
+
 3. Build the player:
 ````bash
 # Build once
-grunt
+npm run build
 # Complete Watch - builds JS, lints, and tests on each change
-grunt serve
-# Quick JS Watch - build only. Requires webpack-dev-server to be installed globally
-webpack-dev-server -w --env.debug --port 8888 --output-public-path /bin-debug/
+npm run start
 # Open the test page from another terminal window
-open http://localhost:8888/test/manual/
+open http://localhost:3000/test/manual/

```

Comment:
hmm currently this doesn't seem to resolve anywhere. Is this something you have to update after that localhost issue during hackweek? @jwbrandon

Reply:
It should be available in this repo (jwplayer open source) after running `npm run start`

Reply:
hmm for some reason I'm running into this error:
<img width="1378" alt="Screen Shot 2022-02-23 at 2 31 21 PM" src="https://user-images.githubusercontent.com/27169173/155420337-68994ad3-9755-4268-98a1-222dc211c768.png">

even though I do have the `karma.conf.js` does exist.

Reply:
whoops! nvm I re-ran after deleting node_modules and package-lock.json